src/tree-selector/treeSelectorComponent.ts

```diff
@@ -0,0 +1,52 @@
+import * as ng from 'angular';
+export class TreeSelectorController {
+
```

Comment:
move this extra newline between the `import` and the `export` :wink: or just simply delete it